robusta_krr/core/runner.py

```diff
@@ -140,7 +139,8 @@ async def _collect_result(self) -> Result:
                 self.warning("Note that you are using the '*' namespace filter, which by default excludes kube-system.")
             return Result(scans=[])
 
-        resource_recommendations = await self._gather_objects_recommendations(objects)
+        with ProgressBar(self.config, total=len(objects), title="Calculating strategy") as progress_bar:
```

Comment:
Maybe to set here `progress_bar` as `self.__progressbar`, so there is no need to pass it through functions.

Reply:
Maybe also to set a title as `"Calculating Recommendation"`